src/Components/Aside/Aside.jsx

```diff
@@ -0,0 +1,19 @@
+import data from "../../stretched-goal.json"
```

Comment:
well done for tackling this stretch goal, super impressed! I really like your playlist, I remember you comented in your demo that putting it in the sidebar would ruin the styling, but i like it as a horizontal bar

Reply:
Thank you. I struggled with this project's styling, so I am glad to hear that.